lib/volcanic/authenticator/v1/scope.rb

```diff
@@ -44,10 +47,36 @@ def to_s
         base
       end
 
+      # For each element in the VRN, assign it a value
+      # element = * (0)
+      # element is value (0.3)
+      # Stack multiply 1
+      # Dataset multiply 2
+      # Resource multiply 3
+      # ResourceId multiply 4
+      # Qualifiers multiply 5
+      def score
+        %i[stack_id dataset_id resource resource_id qualifiers].map do |element|
+          next if send(element).blank?
+
+          if send(element) == '*'
+            0 * multiplier(element)
+          elsif send(element) > '*'
```

Comment:
this is interesting, does the splat operator take on special meaning with the `>` operator?
did a bit of playing around and it basically looks like everything evaluates true but just wondering why you simply didn't do `thing != '*'`

Reply:
Very good point, and on top of that I'm going to refactor out the above issue 👍

Reply:
cool, was just more interested tbh if there was a special case here

Reply:
Regards the splat, it's not actually a splat, just an asterisk as a string 😃

Reply:
indeed it is... 😉